Subject: [PATCH] radeonfb: typos fixes
From: Benjamin Herrenschmidt <benh@kernel.crashing.org>
To: Andrew Morton <akpm@osdl.org>, Linus Torvalds <torvalds@osdl.org>
Cc: Linux Kernel list <linux-kernel@vger.kernel.org>
Date: 	Sun, 13 Feb 2005 15:03:55 +1100

Hi !

The dynamic clock code in radeonfb comes almost as-is from X.org (where
it was contributed by ATI). It has a few typos (wrong register access
macros) that this patch fixes.

Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
Signed-off-by: Olaf Hering <olh@suse.de>

Index: linux-work/drivers/video/aty/radeon_pm.c
===================================================================
--- linux-work.orig/drivers/video/aty/radeon_pm.c	2005-02-13 23:18:52.000000000 +1100
+++ linux-work/drivers/video/aty/radeon_pm.c	2005-02-13 15:01:11.005138528 +1100
@@ -180,7 +180,7 @@
 		tmp = INPLL(pllMCLK_CNTL);
 		tmp &= ~(MCLK_CNTL__FORCE_MCLKA |
 			 MCLK_CNTL__FORCE_YCLKA);
-		OUTREG(pllMCLK_CNTL, tmp);
+		OUTPLL(pllMCLK_CNTL, tmp);
 		radeon_msleep(16);
 	}
 	/* Hrm... same shit, X doesn't do that but I have to */
@@ -404,7 +404,7 @@
 	    ((INREG(CONFIG_CNTL) & CFG_ATI_REV_ID_MASK) < CFG_ATI_REV_A13)) {
 		tmp = INPLL(pllPLL_PWRMGT_CNTL);
 		tmp |= PLL_PWRMGT_CNTL__TCL_BYPASS_DISABLE;
-		OUTREG(pllPLL_PWRMGT_CNTL, tmp);
+		OUTPLL(pllPLL_PWRMGT_CNTL, tmp);
 		radeon_msleep(15);
 	}
 



-
To unsubscribe from this list: send the line "unsubscribe linux-kernel" in
the body of a message to majordomo@vger.kernel.org
More majordomo info at  http://vger.kernel.org/majordomo-info.html
Please read the FAQ at  http://www.tux.org/lkml/

